documentation/book/proc-upgrading-brokers-newer-kafka.adoc

```diff
@@ -4,39 +4,48 @@
 
 [id='proc-upgrading-brokers-newer-kafka-0-9-0-to-0-10-0-{context}']
```

Comment:
I think we changed this largely to a general chapter. We should probably remove the reference to specific versions.

Reply:
Yes. Updated.